test/specs/anInitialSuite.e2e.ts

```diff
@@ -31,29 +31,21 @@ describe('An Initial Suite', async () => {
   });
 
   step('Verify our extensions are not initially loaded', async () => {
-    const workbench = await (await browser.getWorkbench()).wait();
-    await utilities.showRunningExtensions(workbench);
-
-    const extensionNameDivs = await $$('div.name');
-    let sfdxKeywordWasFound = false;
-    let salesforceKeywordWasFound = false;
-    for (const extensionNameDiv of extensionNameDivs) {
-      const text = await extensionNameDiv.getText();
-      if (text.includes('sfdx')) {
-        sfdxKeywordWasFound = true;
-        utilities.log(`AnInitialSuite - extension ${text} was present, but wasn't expected`);
-      } else if (text.includes('salesforce')) {
-        if (text !== 'salesforce.system-tests') {
-          // salesforce.system-tests is expected, anything else is an issue.
-          salesforceKeywordWasFound = true;
-          utilities.log(
-            `AnInitialSuite - extension ${text} was present, but wasn't expected before the extensions loaded`
-          );
-        }
-      }
+    await utilities.showRunningExtensions();
+    await utilities.zoom('Out', 4, 1);
+
+    const foundSfExtensions = await utilities.findExtensionsInRunningExtensionsList(
```

Comment:
The using a new utility that can take an array of extension ids to verify if there are running, thus removing "busy" that duplicates a lot of the functionality in the new function.